imageio/imageio-pict/src/main/java/com/twelvemonkeys/imageio/plugins/pict/PICTImageReader.java

```diff
@@ -1567,10 +1570,13 @@ private void readCompressedQT(final ImageInputStream pStream) throws IOException
         }
 
         // Matrix
-        int[] matrix = new int[9];
+        float[] matrix = new float[9];
         for (int i = 0; i < matrix.length; i++) {
-            matrix[i] = pStream.readInt();
+            int fp = i%3==2 ? 30 : 16; // u v w are 2.30 fixed point, a b c d x y are 16.16 fixed point
+            matrix[i] = pStream.readInt() / (float)(1<<fp);
         }
+        matrix[6] = matrix[6] / (float)screenImageXRatio;
+        matrix[7] = matrix[7] / (float)screenImageYRatio;
```

Comment:
Of curiosity, do you have any sample images that has a ratio other than 1:1? 

It could be that this is correct, I just need some convincing arguments... 😉